Documentation/compatibility/uri-unicode-scheme-only-relative.md

```diff
@@ -0,0 +1,40 @@
+## Support special relative URI notation when Unicode is present
+
+### Scope
+Edge
+
+### Version Introduced
+4.7.3
```

Comment:
Is this a 4.7.2 or a 4.7.3 change?

Reply:
This is supposed to be 4.7.2 I think.

Reply:
That's correct. I'm getting ahead of myself here on version.